src/main/java/com/blockout/controls/Button.java

```diff
@@ -15,6 +16,7 @@
     private static final ResourceLocation soundClick = new ResourceLocation("gui.button.press");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "soundClick" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1068)